lib/rouge/lexers/sql.rb

```diff
@@ -115,7 +115,7 @@ def self.keywords_type
         rule %r/"/, Name::Variable, :double_string
         rule %r/`/, Name::Variable, :backtick
 
-        rule %r/\w[\w\d]*/ do |m|
+        rule %r/\w+/ do |m|
```

Comment:
This is probably a mistake and they meant either `\p{L}\w+` or `[a-zA-Z]\w+`, would have to check the language spec for that.